File: frontend/src/app/models/ClientAutoCreatePlaylists.ts
import { MessageBase, MessageType } from "./MessageBase";
import { PlaylistSortMode } from './PlaylistSortMode';

export class ClientAutoCreatePlaylists extends MessageBase
{
    constructor() {
        super();
    }
    readonly Type : MessageType = MessageType.AutoCreatePlaylists;
    SortMode : PlaylistSortMode;
    MaxPerNamePlaylist : number;
}

File: BeatOn/ClientModels/ClientAutoCreatePlaylists.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using QuestomAssets.BeatSaber;

namespace BeatOn.ClientModels
{
    public class ClientAutoCreatePlaylists : MessageBase
    {
        public override MessageType Type => MessageType.AutoCreatePlaylists;
        public PlaylistSortMode SortMode { get; set; }
        public int? MaxPerNamePlaylist { get; set; }
        public bool RemoveEmptyPlaylists { get; set; }
    }
}